docs/api/en/materials/Material.html

```diff
@@ -407,7 +407,7 @@ <h3>[method:undefined dispose]()</h3>
 			method whenever this instance is no longer used in your app.
 		</p>
 		<p>
-			Material textures must be be disposed of by the dispose() method of
+			Material textures must be disposed by the dispose() method of
```

Comment:
Not 100% sure but I think `disposed of` is right.

Reply:
got it back